spec/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Michael Owen', bio: 'Mike bio', photo: 'some link', postsCounter: 12) }
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'postsCounter should allow valid values' do
+    subject.postsCounter = -2
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Dear @marvin-nyalik, good job adding the `user_spec` test file to test your user model, but we need to keep in mind that any custom methods in the models should have a corresponding unit test. and you are missing to test the `recent_posts` method which is inside the `use` model.

Reply:
Fixed.